assets/airlines/fin.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icao": "fin",
+    "name": "Finnair",
+    "callsign": {
+        "name": "Finnair",
+        "length": 3
+    },
+    "fleets": {
+        "long": [
+            ["A333",  8],
+            ["A359",  8]
+        ],
+        "default": [
+            ["A319",  9],
+            ["A320", 10],
+            ["A321", 16],
+            ["AT72", 11],
+            ["E190", 12]
```

Comment:
Ok but should I then create another fleet using what's currently in default?

I'm under impression that those two types operate almost solely long-haul (mostly to Far East, a few to North America or the nearby holiday destinations at Caribbean, Mexico, etc.) so it would be quite untypical for them to appear to fly from/to certain directions/destinations (the only exception I've heard of is A359 doing HEL-LHR five times a week).